### PR TITLE
Photoshop UXP API documentation update

### PR DESCRIPTION
**Pull request by samgannawayA**
This update fixes a number of small issues, but also adds:

-More layer filters (see Beta section in change log).
-Known Issues section.

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/pages/ps_reference/classes/layer.md [hunk 1]
```diff
@@ -185,7 +229,7 @@ areas, and path to the distortion image map.
 | `verticalScale` | *number* | How much to scale in the direction of height [-999,999]. |
 | `displacementType` | [*DisplacementMapType*](/ps_reference/modules/constants/#displacementmaptype) | Describes how the displacement map fits the image if the image is not the same size as the map. |
 | `undefinedAreas` | [*UndefinedAreas*](/ps_reference/modules/constants/#undefinedareas) | The method used to treat undistorted areas or areas left blank in an image to which the filter in the Distor category has been applied. |
-| `displacementMapFile` | *string* | Path to the distortion image map. |
```
Comment (heewooa) [inline src/pages/ps_reference/classes/layer.md:188]: 👍

File: src/pages/ps_reference/objects/createoptions/layercreateoptions.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ keywords:
 | group | *boolean* | false
  | - | Whether to use previous layer to create clipping mask |
 | mode | [*BlendMode*](/ps_reference/modules/constants/#blendmode) | normal
- | - | Blend mode of the newly created layer. Deprecated, please use [[LayerCreateOptions.blendMode]] as it will override this value. |
+ | - | Blend mode of the newly created layer. Deprecated, please use &#x60;blendMode&#x60; above as it will override this value. |
```
Comment (heewooa) [inline src/pages/ps_reference/objects/createoptions/layercreateoptions.md:32]: This is supposed to be \` (grave)? I didn't know the pipeline currently does not support this correctly...

**Review by heewooa — APPROVED** [review]
Thanks.

File: src/pages/ps_reference/objects/createoptions/layercreateoptions.md (hunk 2 above)
Comment (samgannawayA) [reply src/pages/ps_reference/objects/createoptions/layercreateoptions.md:32]: Yeah, once we make # links for properties, we can revert this change.

**Pull request merged**